pkg/virt-handler/virtwrap/api/schema.go

```diff
@@ -143,6 +168,29 @@ type DomainList struct {
 	Items    []Domain
 }
 
+func (in *DomainList) DeepCopyInto(out *DomainList) {
+	v, _ := model.Clone(in)
```

Comment:
Could you check the error and panic if an error occured?

Reply:
:+1: